web_responsive/static/src/less/main.less

```diff
@@ -100,3 +100,48 @@ main {
 .o_chat_window {
     z-index: 1000;
 }
+
+
+/**
+ *
+ * Vertical Scrollbar
+ *
+ */
+.app-drawer-scrollbar {
+	position:absolute;
+	z-index:100;
+	width:8px;bottom:7px;top:2px;right:1px
+}
+
+.app-drawer-scrollbar > div {
+	position:absolute;
+	z-index:100;
+	width:100%;
+
+	/* The following is probably what you want to customize */
+	background:-webkit-gradient(linear, 0 0, 100% 0, from(#f00), to(#900));
+	background-image:-moz-linear-gradient(top, #f00, #900);
+	background-image:-o-linear-gradient(top, #f00, #900);
```

Comment:
I miss the standard form

Reply:
I'm not following?

Note though, I'll be deleting all of this. It was testing code that I forgot to remove once I figured out how to actually make the scrollbar appear